raiden/transfer/channel.py

```diff
@@ -1429,6 +1584,7 @@ def events_for_expired_lock(
         assert merkletree, "create_sendexpiredlock should return both message and merkle tree"
         channel_state.our_state.merkletree = merkletree
         channel_state.our_state.balance_proof = send_lock_expired.balance_proof
+        channel_state.our_state.nonce = send_lock_expired.balance_proof.nonce
```

Comment:
This makes the update in `create` unnecessary.

Reply:
There's no update, only generate